core-bundle/src/Resources/contao/drivers/DC_Table.php

```diff
@@ -2682,7 +2690,14 @@ public function toggle($intId=null)
 		$objVersions->initialize();
 
 		Input::setPost('FORM_SUBMIT', $this->strTable);
-		$this->varValue = $objRow->{$this->strField};
+		$this->varValue = $this->activeRecord->{$this->strField};
+
+		// TODO: Maybe we don't have to merge manually here anymore if we find a general solution for the save()
+		// problem.
+		$this->denyAccessUnlessGranted(
+			ContaoCorePermissions::DC_PREFIX . $this->strTable,
+			new UpdateAction($this->strTable, $this->activeRecord->row(), array_merge($this->activeRecord->row(), array($this->strField => $this->activeRecord->row())))
```

Comment:
third argument of `UpdateAction` should only contain the updated data to me (maybe that could even be made clearer in the action itself)

Reply:
Nope, it must contain it all.